save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/CloneRepositoryController.kt

```diff
@@ -58,4 +64,47 @@ class CloneRepositoryController(
             .toEntity(String::class.java)
             .toMono()
     }
+
+    /**
+     * Endpoint to save project as binary file
+     *
+     * @param binaryExecutionRequest information about project
+     * @param property file with save properties
+     * @param binaryFile binary project file
+     * @return mono string
+     */
+    @PostMapping(value = ["/submitExecutionRequestBin"], consumes = ["multipart/form-data"])
+    fun submitExecutionRequestByBin(
+        @RequestPart binaryExecutionRequest: BinaryExecutionRequest,
+        @RequestPart("property", required = true) property: Mono<File>,
+        @RequestPart("binFile", required = true) binaryFile: Mono<File>,
+    ): Mono<StringResponse> {
```

Comment:
Do you need `StringResponse` here or can you use a simple String?

Reply:
I think no, because we return `ResponseEntity` and we, probably, want to know status and text in body in response